packages/babel-plugin/package.json

```diff
@@ -30,9 +29,6 @@
     "source-map": "^0.5.7",
     "stylis": "4.1.3"
   },
-  "peerDependencies": {
-    "@babel/core": "^7.0.0"
-  },
```

Comment:
This feels off but I guess that it's not a big deal if we omit it from here - we mostly use Babel through what we are given by the caller anyway.

Reply:
i'm also still curious though - is this removal technically needed to fix the mentioned issues? Isn't inling`@babel/plugin-syntax-jsx` already sufficient?

Reply:
[This comment](https://github.com/emotion-js/emotion/pull/2985#issuecomment-1429222243) indirectly answers this question. This is somewhat needed today because we have a direct dependency on `@emotion/babel-plugin` in some packages.